src/libraries/System.IO.Packaging/src/System/IO/Packaging/InternalRelationshipCollection.cs

```diff
@@ -161,7 +161,7 @@ internal void Flush()
         internal static void ThrowIfInvalidRelationshipType(string relationshipType)
         {
             // Look for empty string or string with just spaces
-            if (relationshipType.Trim() == string.Empty)
+            if (relationshipType.Trim().Length == 0)
```

Comment:
Seems like this should be:
```C#
if (string.IsNullOrWhitespace(relationshipType))
```
to avoid allocating an unnecessary string as part of the trim.

Reply:
```suggestion
            if (string.IsNullOrWhitespace(relationshipType))
```